src/ArchiveManagement/NexusMods.FileExtractor/Extractors/SevenZipExtractor.cs

```diff
@@ -141,7 +142,7 @@ private async Task ExtractAllAsync_Impl(IStreamFactory sFn, AbsolutePath destina
                 .WithStandardOutputPipe(PipeTarget.ToDelegate(line =>
                 {
                     if (string.IsNullOrWhiteSpace(line)) return;
-                    processOutput.Append($"[7z.exe] {line} \n");
+                    processStdOutput.Append($"[7z.exe stdout] {line} \n");
```

Comment:
You can use [`StringBuilder.AppendLine`](https://learn.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.appendline?view=net-7.0#system-text-stringbuilder-appendline(system-string)). It's also probably best to just use `7z` since the App is cross-platform:

```csharp
processStdOutput.AppendLine($"[7z stdout] {line}");

processStdError.AppendLine($"[7z stderr] {line}");
```